tests/one_dragon/test_code_sync_progress.py

```
@@ -0,0 +1,160 @@
+        ('INFO', '检查工作区状态'),
+    ]
+
+
+def test_runtime_launcher_sync_code_uses_framework_log(monkeypatch) -> None:
+    messages: list[str] = []
+
+    class FakeEnvConfig:
+        auto_update = True
+
+    class FakeGitService:
+        instances: list['FakeGitService'] = []
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)